bin/run_brick.rb

```diff
@@ -31,11 +50,14 @@ def get_brick_params(prefix)
   params['gooddata_ruby_commit'] = commit_hash
   params['GDC_LOG_DIRECTORY'] = ENV['GDC_LOG_DIRECTORY'] || '/tmp/'
   params['GDC_EXECUTION_ID'] = execution_id
+  params['GDC_EXECUTION_LOG_PATH'] = ENV['GDC_EXECUTION_LOG_PATH']
+  params['GDC_EXECUTION_RESULT_LOG_PATH'] = ENV['GDC_EXECUTION_RESULT_LOG_PATH']
   log.info "action=#{brick_type}_execution status=start commit_hash=#{commit_hash} execution_id=#{execution_id}"
   brick_pipeline.call(params)
 rescue Exception => e # rubocop:disable RescueException
-	execution_log = GoodData.logger
-	execution_log.error "Execution failed. Error: #{e}" unless execution_log.nil?
+  execution_log = GoodData.logger
+  execution_log.error "Execution failed. Error: #{e}" unless execution_log.nil?
+  log_execution_summary_error(params, log, brick_type, e)
```

Comment:
should not reuse the 'params' because it was modified when execute pipeline.

Reply:
Won't fix. We only get parameters that don't modified during execute pipeline.

Reply:
If you look into the method convert_params in this code base https://github.com/gooddata/gooddata-ruby/blob/master/lib/gooddata/lcm/lcm2.rb#L164, you will see that the params has been modified by the symbolize_keys! method